Automattic/wp-calypso
# Stats: Deprecate the feature flag "stats/new-main-chart"

#### Proposed Changes

* Remove redundant class name `stats--new-wrapper` of the `Main` layout component on the Traffic, Insights, Store, and Ads pages.
* Remove the legacy class name `stats-module` of the ChartTabs wrapper on the `StatsEmailOpenDetail` page.
* Remove all flags usage of `stats/new-main-chart`.

#### Testing Instructions



* Spin up this change with the Calypso Live link.
* Navigate to the pages below:
    * Traffic - `/stats/ads/day/${site}`
    * Insights - `/stats/insights/${site}`
    * Store - `/store/stats/orders/day/${site}` ( [Instructions to display the page ](https://github.com/Automattic/wp-calypso/pull/70253) )
    * Ads - `/stats/ads/day/${site}` ( [Instructions to display the page ](https://github.com/Automattic/wp-calypso/pull/69743) )
* Ensure all layouts work as previously.
* Navigate to the `Email Stats` page - `/stats/email/open/${site}/day/${post-id}?flags=newsletter/stats`. ( [Instructions to display the page ](https://github.com/Automattic/wp-calypso/pull/71548) )
* Ensure the `Chart` loading works fine.

https://user-images.githubusercontent.com/6869813/210797820-429a3a64-1c79-4175-a90e-7ec382377862.mov

#### Pre-merge Checklist





Related to #70209
